source/linux/Foundational_Components/U-Boot/UG-Network-K3.rst

```diff
@@ -102,5 +122,31 @@ binaries files to be sent over TFTP are listed in the table below.
         next-server 10.0.0.1;
       }
 
+.. ifconfig:: CONFIG_part_variant in ('J722S')
+
+  If using ISC dhcpd an example host entry would look like this:
+
+  .. code-block:: text
+
+      subnet 10.0.0.0 netmask 255.0.0.0
```

Comment:
Something tells me this infinite scrolling block of device specific config snippets should be templated at this point.